### PR TITLE
[DA] iss-4887: document resourceids property in event object

### PR DESCRIPTION
**Pull request by dakala**
Issue: https://github.com/fullcalendar/fullcalendar/issues/4887

Not sure how to format **requires premium**

### CODE REVIEW

File: _docs-v4/event-model/event-object.md [hunk 1]
```diff
@@ -169,6 +169,20 @@ A reference to the [Event Source](event-source-object) this event came from. If
 </td>
 </tr>
 
+<tr>
+<th>resourceId</th>
+<td markdown='1'>
+String. The `id` of a [Resource](resource-object) that this event is associated with. (**requires premium**)
```
Comment (arshaw) [inline _docs-v4/event-model/event-object.md]: for the "requires premium" text, can you link to https://fullcalendar.io/docs/premium ?

Comment (dakala) [reply _docs-v4/event-model/event-object.md]: done

**Comment by arshaw** [comment]
The ticket was poorly specified, sorry. There's only a problem with v3. V4 uses the `Event::getResources` method (already documented) to achieve this.

Can you simply transfer these modifications to v3?

**Comment by dakala** [comment]
Moved to v3 as requested.

**Comment by arshaw** [comment]
👍

**Pull request merged**